src/components/Header/HeaderContact.js

```diff
@@ -90,12 +90,16 @@ const styles = StyleSheet.create({
   }),
   text: {
     color: COLORS.black,
-    fontFamily: fonts.inter[600],
+    fontFamily: fonts.inter[500],
     fontSize: normalizeFontSize(14),
-    fontWeight: 'bold',
     padding: 10,
     textAlign: 'center'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the text style properties, specifically the fontFamily and fontWeight, may alter the appearance of the text. Make sure these changes align with the overall design language of your application.
```diff
-    fontFamily: fonts.inter[600],
-    fontWeight: 'bold',
+    fontFamily: fonts.inter[500],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->